source/index.html.md.erb

```diff
@@ -3,21 +3,24 @@ title: Documentation – GOV.UK Pay
 weight: 10
 ---
 
-# GOV.UK Pay documentation
+# GOV.UK Pay technical documentation
 
-Use the left-hand navigation bar to go to different sections of the
-documentation.
+Use this technical documentation to find out how to:
 
-You can also read more about [what GOV.UK Pay offers](https://www.payments.service.gov.uk/features).
+- use the GOV.UK Pay API
+- integrate your service with GOV.UK Pay
 
-Please [contact us](/support_contact_and_more_information/#contact-us) if you have any questions or feedback.
+You can read [non-technical information](https://www.payments.service.gov.uk/using-govuk-pay/) about:
 
-## Services using GOV.UK Pay
+- creating an account and selecting a Payment Service Provider (PSP)
+- managing your account with Stripe, GOV.UK Pay's PSP
+- setting up payment links
+- security and PCI compliance
 
-You can [see how many services are using GOV.UK Pay](https://www.gov.uk/performance/govuk-pay#number-services-heading).
+[Contact us](/support_contact_and_more_information/#contact-us) if you have any questions or feedback.
 
 ## How much it costs
 
-There is no charge to use GOV.UK Pay.
+There's no charge to use the GOV.UK Pay API.
 
-You will still need to pay your payment service provider’s (PSP’s) transaction fees. If you use GOV.UK Pay's PSP, we'll automatically deduct the transaction fee from each payment.
+You'll still need to pay your payment service provider’s (PSP’s) transaction fees. If you use GOV.UK Pay's PSP, we'll automatically deduct the transaction fee from each payment.
```

Comment:
this still feels like a bit of an add on, but I think we leave it as is for now and come back to it later...

Reply:
Yes agree - I feel like it might be in the wrong place. But yep let's leave for now and I'll make a note to look at this when we look at payment flow.